src/mongo_client.ts

```diff
@@ -416,10 +407,6 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> {
     return this.s.bsonOptions;
   }
 
-  get logger(): LegacyLogger {
```

Comment:
this and the getLogger method in L710 need to get added to the deprecation ticket